website/src/pages/en/indexing/tap.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: TAP Migration Guide
+title: GraphTally Guide
 ---
 
-Learn about The Graph’s new payment system, **Timeline Aggregation Protocol, TAP**. This system provides fast, efficient microtransactions with minimized trust.
+Learn about The Graph’s new payment system, **GraphTally**. This system provides fast, efficient microtransactions with minimized trust.
 
 ## Overview
 
-[TAP](https://docs.rs/tap_core/latest/tap_core/index.html) is a drop-in replacement to the Scalar payment system currently in place. It provides the following key features:
+[GraphTally](https://docs.rs/tap_core/latest/tap_core/index.html) is a drop-in replacement to the Scalar payment system currently in place. It provides the following key features:
```

Comment:
There is zero mention of "GraphTally" at that URL, so perhaps it needs some additional context?

Reply:
Good call - fixed that!

Reply:
Awesome, great work.